app/gui/controller/engine-protocol/src/language_server/tests.rs

```diff
@@ -1,14 +1,14 @@
-use std::future::Future;
+use super::*;
```

Comment:
We should not use `super::*` in modules being in a separate file. https://github.com/enso-org/enso/blob/f233ea66b37969b84a224e462014853f00db285e/app/gui/docs/contributing/style-guide.md#submodules-and-imports so let's fix it on this opportunity.